src/pal/src/exception/seh-unwind.cpp

```diff
@@ -491,7 +491,17 @@ BOOL PAL_VirtualUnwindOutOfProc(CONTEXT *context,
 
     LibunwindCallbacksInfo.Context = context;
     LibunwindCallbacksInfo.readMemCallback = readMemCallback;
+#if UNWIND_CONTEXT_IS_UCONTEXT_T
     WinContextToUnwindContext(context, &unwContext);
+#else
+    st = unw_getcontext(&unwContext);
```

Comment:
I am not sure how this would work - instead of using the supplied context that the function is expected to unwind, you get the current context of the current thread which is completely different.